diff --git a/stubs/urlfetch.pyi b/stubs/urlfetch.pyi
index c694df8a..66e103ae 100644
--- a/stubs/urlfetch.pyi
+++ b/stubs/urlfetch.pyi
@@ -80,14 +80,14 @@ class Session:
 def fetch(*args: Any, **kwargs: Any): ...
 def request(url: Any, method: str = ..., params: Optional[Any] = ..., data: Optional[Any] = ..., headers: Any = ..., timeout: Optional[Any] = ..., files: Any = ..., randua: bool = ..., auth: Optional[Any] = ..., length_limit: Optional[Any] = ..., proxies: Optional[Any] = ..., trust_env: bool = ..., max_redirects: int = ..., source_address: Optional[Any] = ..., validate_certificate: Optional[Any] = ..., **kwargs: Any): ...
 
-get: Any
-post: Any
-put: Any
-delete: Any
-head: Any
-options: Any
-trace: Any
-patch: Any
+GET: str
+POST: str
+PUT: str
+DELETE: str
+HEAD: str
+OPTIONS: str
+TRACE: str
+PATCH: str
 
 class ObjectDict(dict):
     def __getattr__(self, name: Any): ...
